posthog/queries/test/test_trends.py

```diff
@@ -318,7 +318,7 @@ def test_breakdown_filtering(self):
 
             self.assertEqual(sum(response[0]["data"]), 2)
             self.assertEqual(response[0]["data"][4 + 7], 2)
-            self.assertEqual(response[0]["breakdown_value"], "None")
```

Comment:
This will possibly mess stuff up in the frontend?

Reply:
I haven't found any comparison to `'None'` in the frontend code base, so think this shouldn't affect it (unless I'm missing some value handling somewhere).

Reply:
appUrlsLogic.js:45 relies on it being true/false (rather than 'nan'), same for trendsLogic.js:100.

Reply:
This looks like it was a truthy string anyway though, as in `"None"`, not `None`.